docs/administration.rst

```diff
@@ -262,22 +262,101 @@ and are used when Lemur creates the CSR for your certificates.
         LEMUR_DEFAULT_AUTHORITY = "verisign"
 
 
+.. _NotificationOptions:
+
 Notification Options
 --------------------
 
-Lemur currently has very basic support for notifications. Currently only expiration notifications are supported. Actual notification
-is handled by the notification plugins that you have configured. Lemur ships with the 'Email' notification that allows expiration emails
-to be sent to subscribers.
+Lemur supports a small variety of notification types through a set of notification plugins.
+By default, Lemur configures a standard set of email notifications for all certificates.
+
+**Plugin-capable notifications**
+
+These notifications can be configured to use all available notification plugins.
+
+Supported types:
+
+* Certificate expiration
+
+**Email-only notifications**
+
+These notifications can only be sent via email and cannot use other notification plugins.
+
+Supported types:
+
+* CA certificate expiration
+* Pending ACME certificate failure
+* Certificate rotation (currently disabled in code)
```

Comment:
shall we change this to "currently not enabled in the celery task for rotation"?

one could create a new celery task which indeed sends out notifications: 
https://github.com/Netflix/lemur/blob/e87cf040f3fa531f407a34876ae71b4970d6369f/lemur/common/celery.py#L661-L663


or 

make sure the cron task includes the notify option
```certificate rotate -c -a```

https://github.com/Netflix/lemur/blob/56169656377637b7a0e53514105fbe31b436c4a9/lemur/certificates/cli.py#L186-L192

Reply:
I'll change the docs, but I'm also going to add a config option that allows celery to send rotation notifications without a code change.